docs/onboard-public-officers-using-non-se-machines.md

```diff
@@ -55,19 +42,17 @@ This article guides how public officers can get a TechPass account:
 
 ## Step 2. Sign up for TechPass
 
-> **Notes**:
->- If you are a GCC 1.0 user and activating your TechPass account based on instructions on this page, skip this step as you would have registered for TechPass as mentioned earlier in this page.
->- Other users need to complete the below steps using a non-SE GSIB machine.
+> **Note**:
+> If you are a public officer using a SE-GSIB device, create a [service request](https://go.gov.sg/techpass-sr) to get your TechPass account.
 
 <details>
-  <summary style="font-size:18px">How to sign up for TechPass?</summary>
-
-  Public officers sign up for their TechPass account using their official email address. An invitation link will be sent to this email address for them to accept.
-
-?> Note: As LiteMail accounts are not supported, upgrade to a standard mailbox before signing up for TechPass. Format of a standard, official email address of a public officer will be *your_name<span>@</span>agency.gov.sg*
+  <summary style="font-size:20px;font-weight:bold">How to sign up for TechPass?</summary>
 
+  - Public officers need to sign up for their TechPass account using their official email address. An invitation link will be sent to this email address for them to accept.
+  - Public officers must use their official email address. Format of a standard, official email address of a public officer is *your_name<span>@</span>agency.gov.sg*.
+  - LiteMail accounts are not supported by TechPass. If you have a LiteMail account, upgrade it to a standard mailbox before signing up for TechPass.
```

Comment:
i'm not sure if there's other cases where an email address has no inbox. Thinking whether we should turn the messaging around so that we say the email address has to have an inbox, then for example LiteMail accounts are not supported.

Reply:
TechPass does not support email accounts such as LiteMail accounts, which do not have an inbox. If you use such an email account, upgrade it to a standard mailbox before signing up for TechPass. 

@JLeeChan is this technically correct, and does it cover all such use cases?

Reply:
Done!